linux_priv.c

```diff
@@ -3,10 +3,51 @@
 #include <errno.h>
 #include <stdlib.h>
 #include <sys/ioctl.h>
+#include <signal.h>
+#include <string.h>
 #include "memcached.h"
 
+static char *kill_msg;
+#define KILL_MSG_STR "Seccomp policy failure. Caught syscall ???. This is either an exploit attempt, or your system is not supported yet.\n"
```

Comment:
split line?

Reply:
also, add clear warning comment that the code below is directly indexing into it, please!